tests/source/nested_modules.rs

```diff
@@ -8,12 +10,23 @@
 //      "nested_modules::example_module::nested"
 pub mod example_module {
```

Comment:
@euclio i would also like to add a test above, to show that this module does *not* have a parent link. I tried to use `@!has` for this, but couldn't quite get it, any thoughts?

Reply:
Something like

```
// @matches "included[?id=='nested_modules::example_module'] \
//          .relationships.parent" []
```

Should do the trick.

Reply:
ah, nice